### PR TITLE
add links to p2pool block explorer

### PR DESCRIPTION
**Pull request by SChernykh**
All explorers listed on the page don't have Tor Onion Service (aka hidden service) available, so I added P2Pool's explorer.

### CODE REVIEW

**Review by selsta — APPROVED** [review]

File: resources/tools/index.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ permalink: /resources/tools/index.html
                         <p><a href="https://moneroblocks.info">MoneroBlocks</a></p>
                         <p><a href="https://www.exploremonero.com/">Explore Monero</a></p>
                         <p><a href="https://monerovision.com">MoneroVision</a></p>
+                        <p><a href="https://p2pool.io/explorer/">P2Pool</a> (with <a href="http://yucmgsbw7nknw7oi3bkuwudvc657g2xcqahhbjyewazusyytapqo4xid.onion/explorer/">Onion Service</a>)</p>
```
Comment (erciccione) [inline resources/tools/index.md:25]: The second part of the string needs to be adapted  to be multilingual or it will be displayed only in English.

**Review by erciccione — CHANGES_REQUESTED** [review]

File: resources/tools/index.md (hunk 1 above)
Comment (SChernykh) [reply resources/tools/index.md:25]: Do I need to add it to all languages in https://github.com/monero-project/monero-site/blob/master/_i18n/ ? I can't translate it all.

**Review by erciccione — CHANGES_REQUESTED** [review]
~~This seems to have been added to #2107, but that pr has been opened after. Could you clarify the situation and/or get rid of the duplicate pr?~~

Disregard, i see now that this commit was included in  #2107, but it's not noted anywhere as far as i can tell. We can review and merge this PR.

File: _i18n/ru.yml [hunk 2]
```diff
@@ -1753,3 +1753,4 @@ tools:
     пожалуйста,
   intro2: откройте соответствующий вопрос (issue) на GitHub
   monerologs: Архив IRC-каналов Monero
+  onion-service: Скрытая служба Tor
```
Comment (erciccione) [inline _i18n/ru.yml:1756]: Please remove as it could create conflicts with Weblate.

**Review by selsta — APPROVED** [review]

**Review by erciccione — APPROVED** [review]

**Pull request merged**